demo/kserve/Kserve.md

```diff
@@ -6,6 +6,8 @@
 - CLI tools
   - oc cli
 
+- http2 enabled in the cluster
```

Comment:
Please add this too.
``` 
cluster-admin permission need
```

Reply:
don't we need cluster admin permission for other steps too? like installing operators

Reply:
done!

Reply:
It is because of SMMR but it might need less power but just in case, it should be good to have strong permission.

Reply:
no I mean to say that for running the entire quick start we should have the cluster admin permissions. Should we write it in the pre-requiresites rather?

Reply:
In order to install operators, yes cluster admin permission is needed but I realized some users have a cluster all required operators were installed. So I think it is better to write this down into pre-requisites.

Reply:
done. However, we should document it more precisely in the API documentation which I strongly suggest to start authoring